x-pack/plugins/session_view/public/components/Process/index.tsx

```diff
@@ -200,23 +205,23 @@ function Process({ process, isSessionLeader = false, depth = 0, onProcessSelecte
   }
 
   // TODO: not customizable for now (cmd previously offered a template string to render)
-  const template = (process: IProcess) => {
-    const event = process.getLatest();
-    const { args, working_directory } = event.process;
-    const { searchMatched } = process;
+  const template = (templateProcess: IProcess) => {
+    const { args, working_directory: workingDirectory } = templateProcess.getLatest().process;
+    const { searchMatched } = templateProcess;
 
     if (searchMatched !== null) {
       const regex = new RegExp(searchMatched);
 
-      //TODO: should we allow some form of customization via settings?
-      let text = `${working_directory} ${args.join(' ')}`;
+      // TODO: should we allow some form of customization via settings?
+      let text = `${workingDirectory} ${args.join(' ')}`;
```

Comment:
yea i was thinking about this.. Was wondering if there was a cleaner way. If only we just did camelCase in ECS too ha

Reply:
yeah, I was also wondering if there was a cleaner way too, another way could be using it without destructuring the object:

![image](https://user-images.githubusercontent.com/19270322/139718518-b0e2f460-786f-4c35-aa8b-173d62570584.png)

 
but also, I've seen some cases just declaring the camelized variables, (sometimes using the `get` from `lodash`)

![image](https://user-images.githubusercontent.com/19270322/139716876-a370744a-aa88-44d5-93c6-576db3caf0e5.png)